src/rocksdb.rs

```diff
@@ -2543,6 +2543,17 @@ impl Env {
         }
     }
 
+    // Create an cloud env to operate with AWS S3.
+    #[cfg(feature = "cloud")]
+    pub fn new_aws_env() -> Env {
```

Comment:
can you add a test that merely calls this method (as we don't have a valid aws account to test with)? For that test to compile you probably want to update travis.yml to install aws sdk.